### PR TITLE
changed code to remove old stuff

### PR DESCRIPTION
**Pull request by ArjanSaha0108**

### CODE REVIEW

File: src/components/pages/Registration.tsx [hunk 1]
```diff
@@ -115,7 +107,7 @@ export const Registration = () => {
   );
 };
 
-const ticketList: Ticket[] = [
+/*const ticketList: Ticket[] = [
```
Comment (aritraChatterjee) [inline src/components/pages/Registration.tsx]: No commented code please. If you want to retain the code fragment then switch off this component using a flag.

File: src/components/pages/Registration.tsx [hunk 2]
```diff
@@ -126,9 +118,9 @@ const ticketList: Ticket[] = [
     },
     link: 'https://usuc.yapsody.com/event/book/803521/31648557'
   }
-];
+];*/
 
-const Tickets = (props: any) => {
+/*const Tickets = (props: any) => {
```
Comment (aritraChatterjee) [inline src/components/pages/Registration.tsx]: No commented code please. If you want to retain the code fragment then switch off this component using a flag.

File: tsconfig.json [hunk 3]
```diff
@@ -14,7 +18,9 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "preserve"
```
Comment (aritraChatterjee) [inline tsconfig.json:21]: why this change?

**Comment by aritraChatterjee** [comment]
image

image


Please resolve the compilation issue

**Review by aritraChatterjee — CHANGES_REQUESTED** [review]

**Review by ArjanSaha0108 — COMMENTED** [review]
Perfect

**Review by ArjanSaha0108 — COMMENTED** [review]
Looks good

**Review by ArjanSaha0108 — COMMENTED** [review]
Looks good

**Pull request merged**